### PR TITLE
Add Categories to CAPX VMs

### PR DESCRIPTION
**Pull request by yannickstruyf3**
**What this PR does / why we need it**:
- Creates a specific CAPX cluster category in PC
- Adds a default category to all CAPX VMs
- Allows adding additional categories to the VMs

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
N/A

**Special notes for your reviewer**:
Categories will be assigned to the VMs in PC. The category format is: 
kubernetes-io-cluster-<cluster-name>: owned
Steps to test:
- Create CAPX cluster
- Go to PC > VMs
- Click on one of the deployed CAPX VMs
- Go to Categories

Screenshot 2022-04-15 at 20 02 21

**Release note**:

```release-note
- Creates a specific CAPX cluster category in PC
- Adds a default category to all CAPX VMs
- Allows adding additional categories to the VMs
```

### CODE REVIEW

**Review by tuxtof — APPROVED** [review]

**Comment by deepakm-ntnx** [comment]
@yannickstruyf3 could you please add screenshots or kubectl output to show where these categories show up and how can they be used by customers?

**Comment by yannickstruyf3** [comment]
@deepakm-ntnx I added the steps performed to check if the categories are assigned.

File: controllers/helpers.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	infrav1 "github.com/nutanix-core/cluster-api-provider-nutanix/api/v1beta1"
 	nutanixClient "github.com/nutanix-core/cluster-api-provider-nutanix/pkg/client"
 	nutanixClientV3 "github.com/nutanix-core/cluster-api-provider-nutanix/pkg/nutanix/v3"
+	v3 "github.com/nutanix-core/cluster-api-provider-nutanix/pkg/nutanix/v3"
```
Comment (deepakm-ntnx) [inline controllers/helpers.go:27]: line 26 has the same import. can we use the same?

**Review by deepakm-ntnx — APPROVED** [review]
beside one comment above, lgtm

**Review by tuxtof — APPROVED** [review]

**Review by deepakm-ntnx — APPROVED** [review]

**Pull request merged**